Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -551,6 +551,10 @@ void PPCAnalyzer::SetInstructionStats(CodeBlock* block, CodeOp* code, const Gekk
   code->outputFPRF = (opinfo->flags & FL_SET_FPRF) != 0;
   code->canEndBlock = (opinfo->flags & FL_ENDBLOCK) != 0;
 
+  // TODO: Is it possible to determine that some FPU instructions never cause exceptions?
+  code->canCauseException =
+      (opinfo->flags & (FL_LOADSTORE | FL_USE_FPU | FL_PROGRAMEXCEPTION)) != 0;
```

Comment:
The first FPU instruction will throw an exception, so all others are fine.

Reply:
Can't FPU instructions throw exceptions in other situations as well (that may not have been implemented in the JITs yet), like division by zero? (See the discussion about True Crime on IRC earlier this month)